src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -652,4 +661,25 @@ public function get_theme_items( $request ) {
 
 		return $response;
 	}
+
+	/**
+	 * Validate style.css as valid CSS.
+	 *
+	 * Currently just checks for invalid markup.
+	 *
+	 * @since 6.2.0
+	 *
+	 * @param string $css CSS to validate.
+	 * @return true|WP_Error True if the input was validated, otherwise WP_Error.
+	 */
+	private function validate_custom_css( $css ) {
+		if ( preg_match( '#</?\w+#', $css ) ) {
+			return new WP_Error(
+				'rest_custom_css_illegal_markup',
+				__( 'Markup is not allowed in CSS.', 'gutenberg' ),
+				array( 'status' => 400 )
+			);
+		}
+		return true;
+	}
 }
```

Comment:
Also, not advocating abstracting the method, just the `preg_match` part that validates CSS that could be useful outside of this specific controller.  Example, the Customizer.

Reply:
Do you have any thoughts on the best place to put a helper function like this?

Reply:
Proposal:
Keep it here for this backport PR. But open a Trac ticket to explore creating a global reusable function for validating custom CSS.

Since this method is `private` within the class, there's no risk for BC breaks or OOP leaks by abstracting it out of the class later.

In this way, further use cases can be explored to potentially refine the functionality for the needs beyond this class.